cms/models/pluginmodel.py

```diff
@@ -250,6 +250,23 @@ def get_plugin_instance(self, admin=None):
         self._inst = instance
         return self._inst, plugin
 
+    def get_bound_plugin(self, admin=None):
```

Comment:
Ok, then how about the other way around? Using ``get_bound_plugin`` in ``get_plugin_instance`` instead of these lines https://github.com/divio/django-cms/blob/b91e58e14c47a00adcf102851a0a51de5d4de544/cms/models/pluginmodel.py#L239-L250
Having two method each one the subset of the other with slightly different implementations can be confusing and it can generate errors